src/NonPhpFile/NonPhpFileProcessor.php

```diff
@@ -46,6 +54,9 @@ public function __construct(
     public function process(array $files): void
     {
         foreach ($files as $file) {
+            if ($this->skipper->shouldSkipFileInfo($file->getSmartFileInfo())) {
+                continue;
+            }
             $this->processFile($file);
```

Comment:
:+1:

Reply:
After looking at the code again, i see a duplication of this pattern in every FileProcessor. I am trying to move this one level up into the ApplicationFileProcessor.